### PR TITLE
Development env improvements.

### LINKED ISSUE #15: Package and Yarn lock exists on the same project
Is having both lockfiles for `npm` and `yarn` deliberate?

### PR DESCRIPTION
**Pull request by janzenz**
Closes #15 
- [x] Sticking with `yarn`
- [x] Locking node version using `nvm`
- [x] Update Readme for development tool

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,11 @@ Use Comic Sans in Slack! (sorta)
 
 Head over here for more info and to use it:
 https://liamjohnston.github.io/comic-slack/
+
+
+## Development
+```
+nvm use
+yarn install
```
Comment (liamjohnston) [inline README.md]: actually you only need `yarn` not `yarn install` 🤓

**Review by liamjohnston — APPROVED** [review]
Minor thing you could change but fine either way. Thanks!

**Comment by liamjohnston** [comment]
This project also needs a `.prettierrc` probably... if you're in the mood. Could be a separate PR.

**Comment by janzenz** [comment]
@liamjohnston done!

File: package.json [hunk 2]
```diff
@@ -4,11 +4,9 @@
   "homepage": "https://liamjohnston.github.io/comic-slack/",
   "private": true,
   "dependencies": {
-    "node-sass": "^4.12.0",
     "react": "^16.3.2",
     "react-copy-to-clipboard": "^5.0.1",
     "react-dom": "^16.3.2",
-    "react-scripts": "3.2.0",
```
Comment (liamjohnston) [inline package.json]: Any idea why this is removed? This is part of create-react-app, right...? And `node-sass`?

Comment (janzenz) [reply package.json]: Semantically they should be in `dev` but looks like the convention is that these packages need to be in `dependencies` I've just reverted that.

**Review by liamjohnston — APPROVED** [review]

**Comment by liamjohnston** [comment]
Nice work, thanks! May the swag gods rain laptop stickers down on you.

**Pull request merged**